pkg/karmadactl/cmdinit/kubernetes/deployments.go

```diff
@@ -59,7 +59,11 @@ func (i *CommandInitOption) etcdServers() string {
 }
 
 func (i *CommandInitOption) karmadaAPIServerContainerCommand() []string {
-	return []string{
+	var etcdServers string
+	if etcdServers = i.ExternalEtcdServers; etcdServers == "" {
+		etcdServers = strings.TrimRight(i.etcdServers(), ",")
```

Comment:
Multiple IPS are allowed if the extended etcd ip is not empty? Is empty, only one can be used?

Reply:
> Multiple IPS are allowed if the extended etcd ip is not empty?

Yes, this `ExternalEtcdServers` is assigned to `--etcd-servers` arg of apiserver, at the code you point out, it's not care whether multi or one, just assigned to --etcd-servers.

> Is empty, only one can be used?

No. If `ExternalEtcdServers` is empty, it means not specified external etcd servers, it fallbackt to use the original bundled etcd logic. (line `71`)

https://github.com/karmada-io/karmada/blob/9cea80027bd5eedd8277d33cf081918e378a2f69/pkg/karmadactl/cmdinit/kubernetes/deployments.go#L70-L73